srcCxx/shape_main.cxx

```diff
@@ -16,6 +16,7 @@
 #include <string.h>
 #include <stdarg.h>
 #include <iostream>
+#include <getopt.h>
```

Comment:
Need the include where `useconds_t` is declared

```suggestion
#include <getopt.h>
#include <sys/types.h>
```

Reply:
done